src/priorityq.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding utf-8 -*-
+
+
+class Priorityq(object):
+
+    def __init__(self, iterable=None):
+        '''Initialise a Priorityq from a list of lists or tuples
+            where the zeroeth element is the priority'''
+        if iterable is None:
```

Comment:
If most of your code is expecting `self.priorityq` to be a list, it is way more sane and less risky to initialize `self.priorityq` as an empty list in the case where the Priorityq is initialized without any args.